deepmd/pt/model/atomic_model/pairtab_atomic_model.py

```diff
@@ -154,6 +169,65 @@ def deserialize(cls, data) -> "PairTabAtomicModel":
         tab_model.register_buffer("tab_data", torch.from_numpy(tab_model.tab.tab_data))
         return tab_model
 
+    def compute_output_stats(
+        self,
```

Comment:
I am not sure about this. @wanghan-iapcm

Reply:
1. At the atomic_model level, one needs to implement the `compute_or_load_stat`. 
2. We may need to add the abstract method to the `atomic_model`. 
3. The torch implementation of energy stat seems to be abstracted to some level rather than copying the code.